src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/bootstrap-import-modal/bootstrap-import-modal.component.ts

```diff
@@ -106,12 +113,13 @@ export class BootstrapImportModalComponent extends BaseModal implements OnInit,
   validatePools(): ValidatorFn {
     return (poolsControl: UntypedFormGroup): { [key: string]: any } => {
       let checkedCount = 0;
+      console.log(poolsControl.controls)
       _.each(poolsControl.controls, (control) => {
         if (control.value === true) {
           ++checkedCount;
         }
       });
-
+     
```

Comment:
@prgoel-code make sure we run `npm run fix` and `npm run lint` to catch minor issues like this.
```suggestion
 
```